front/src/views/recovery-password/recovery-password.view.js

```diff
@@ -1,47 +1,21 @@
-import { MmtTextInput } from '@components';
+import {
+  MmtTextInput,
+  MmtRecoveryStep1,
+  MmtRecoveryStep2,
+  MmtRecoveryStep3
+} from '@components';
 export default {
   components: {
-    MmtTextInput
+    MmtTextInput,
+    MmtRecoveryStep1,
+    MmtRecoveryStep2,
+    MmtRecoveryStep3
   },
   data () {
     return {
-      email: '',
-      step: 1,
-      password: '',
-      passwordConfirmation: '',
-      valid: false,
-      showPassword: false,
-      showPasswordConfirmation: false,
-      serverError: '',
-      rules: {
-        isEmpty: v => !!v || this.$t('recoveryPassword.noEmpty'),
-        minPassLen: v => v.length >= 8 || this.$t('recoveryPassword.restriction'),
-        validEmail: v => /.+@.+\..+/.test(v) || this.$t('recoveryPassword.thisNoEmail'),
-        samePass: () => this.password === this.passwordConfirmation
-          || this.$t('recoveryPassword.passwordMustIdentity')
-      }
+      step: 1
```

Comment:
hm...i don't think that it is vue way. try to add custom event for your step components and handlers for it.

like 
`<mmt-recovery-step-1 @stepDone="handleStep({stepNumber: 1})"`
`<mmt-recovery-step-2 @stepDone="handleStep({stepNumber: 2})"`

Reply:
just think,  might some events or properties could make your code easier